packages/ast-utilities/src/matchers.ts

```diff
@@ -7,7 +7,7 @@ import {
 declare global {
   // eslint-disable-next-line @typescript-eslint/no-namespace
   namespace jest {
-    interface Matchers<R> {
+    interface Matchers<R, T = {}> {
```

Comment:
Updated to match https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jest/index.d.ts#L697